### DISCUSSION TITLE
Frage zur Installation

### DISCUSSION THREAD

**Discussion by Mattes0101** [camueller/SmartApplianceEnabler#245] [Q&A]
Hallo zusammen,
ich bin ebenfalls grade dabei mit den SAE aufzusetzen.
Dazu habe ich einen Raspi3 V1.2 mit 16GB MicroSD Karte und USB Stick (Fat32) entsprechend der Anleitung hier:
https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Installation_DE.md
aufgesetzt.
Als Image habe ich das letzte ([2022-01-28-raspios-bullseye-armhf-lite-usbrun.zip]) genommen.
Start vom Raspi mit SD Karte läuft wie ich das kenne (habe einen Bildschirm und Maus/Tastatur) abgeschlossen um alles zu sehen.
D.h. Code rennt und wird installiert.
Am Ende bekomme ich eine Fehlermeldung zur Spannungsversorgung ('under-voltage detected' anderes Netzteil ist bestellt). Und es wird die Login-Aufforderung angezeigt (raspberrypi login :). 
Wenn ich den USB Stick dann reinstecke passiert leider garnichts, außer das die LEDs etwas blinken. Es beginnt aber keine installation.

Ich habe auch schon versucht mich mit dem user 'pi' und passwort 'raspberry' [Achtung amerikanisches Tastaturlayout] anzumelden und das dann zu versuchen. Ging leider auch nicht.

Zudem habe ich es per WLAN über die SSID und WPA Key in der Config sowie auch mit Kabel direkt an der Fritzbox versucht. Hat alles nicht geklappt.

Hat evtl. einer eine Idee was ich falsch mache?

LG
Mattes

**Comment by Trunks1982** [comment]
Du hast dir nen image auf die sd karte installiert?

**Reply by Mattes0101** [reply]
Ja, genau nach der Anleitung mit Raspberry Pi Imager und dem o.g. Image sowie einem USB MicroSD Kartenleser. 
Sieht irgendwie koisch aus, weil die SD Karte in einen boot Teil (252MB) und einen nicht formatierten Teil (bzw. mit Windoes nicht lesbaren Teil) formatiert wurde. Ich vermut aber mal, dass das so muss, oder?

**Reply by Trunks1982** [reply]
ich würde ein ganz normales Image Flashen und die manuelle Installation wählen.Ist sozusagen nur nen copy paste.Funktioniert garantiert.

**Reply by Mattes0101** [reply]
Was meinst du mit normalem image?
Also normales 'Betriebssystem' und nicht das hier verlinkte? Und dann analog der Vorgehensweise hier https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/ManualInstallation_DE.md vorgehen?

**Reply by Trunks1982** [reply]
Doch das kannste nehmen.

**Reply by Mattes0101** [reply]
So hat geklappt :-) War zwar viel getippe, aber es läuft. LG
